src/lib/hooks/useDocument.ts

```
@@ -1,9 +1,9 @@
 import { useCallback, useContext, useState } from 'react'
-import { FrappeConfig, FrappeDoc, GetDocListArgs, Filter, FrappeError as Error } from '../types'
+import { FrappeConfig, FrappeDoc, GetDocListArgs, Filter, FrappeError as Error, GetCountResponse } from '../types'
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)